lizmap/lizmap-modules/localiz/install/install.php

```diff
@@ -0,0 +1,49 @@
+<?php
+/**
+* @package   lizmap
+* @subpackage localiz
+* @author    aeag
+* @copyright 2011 3liz
+* @link      http://3liz.com
+* @license    All rights reserved
+*/
+
+
+class localizModuleInstaller extends jInstallerModule {
+
+    function install() {
+        // installer le CSS
+        $cssFile = jApp::wwwPath('css/localiz.css');
+        if (! file_exists($cssFile)) {
+            $this->copyFile('css/localiz.css', $cssFile);
+        }
+        
+        // installer le JS
+        $jsFile = jApp::wwwPath('js/localiz.js');
+        if (! file_exists($jsFile)) {
+            $this->copyFile('js/localiz.js', $jsFile);
+        }
+
+        // conf DB
+        $localConfFile = jApp::configPath('localconfig.ini.php');
```

Comment:
You already have a such instantiated object in `$this->entrypoint->localConfigIni`